Item/Item.module.less

```diff
@@ -16,13 +63,49 @@
 		line-height: @agate-item-line-height;
 		color: @agate-item-color;
 
+		.itemContent {
+			.label   {
+				color: @agate-item-label-color;
+				font-size: @agate-item-label-font-size;
+			}
+		}
+
+		&.active {
+			color: @agate-item-active-color;
+		}
+
+		&.selected {
+			outline: @agate-item-selected-outline;
```

Comment:
I noticed a faint gap in Chrome with device emulation when using `outline`. `box-shadow` doesn't seem to produce the same result so it might be a layout-preserving alternative.

![image](https://user-images.githubusercontent.com/788456/65538459-35d6d380-debc-11e9-98a9-ff164aba49de.png)

Reply:
Was this gap visible on a non-retina display? Was scaling set to 100% (or something else)?

Reply:
Doesn't seem to repro at 100% scaling.

Reply:
My opinion would be to ignore this in that case.